wgpu/src/backend/web.rs

```diff
@@ -553,7 +553,7 @@ fn map_texture_format(texture_format: wgt::TextureFormat) -> web_sys::GpuTexture
     }
 }
 
-fn map_texture_format_from_web_sys(
+fn _map_texture_format_from_web_sys(
```

Comment:
Should this function be removed now that it's unused?

Reply:
Yes